projects/vdk-plugins/vdk-server/src/vdk/plugin/server/installer.py

```diff
@@ -646,6 +646,8 @@ def __helm_install_command(self, git_server_ip):
             "--set",
             "deploymentBuilderResourcesDefault.limits.cpu=0",
             "--set",
+            "dataJob.executions.logsUrl.urlTemplate=http://localhost:8092/data-jobs/for-team/{{team_name}}/jobs/{{job_name}}/executions/{{execution_id}}/logs?tail_lines=400",
```

Comment:
Why is that needed?

Reply:
in quickstart vdk we want ppl to be able to see the logs as provided by our api?

Reply:
nit: it just seems we have a lot of hardcoded duplicates of "http://localhost:8092" strings